cmd/core/main.go

```diff
@@ -230,7 +232,7 @@ func main() {
 	}
 	ctrl.SetLogger(klogr.New())
 
-	leaderElectionID := util.GenerateLeaderElectionID(types.KubeVelaName, controllerArgs.IgnoreAppWithoutControllerRequirement)
+	leaderElectionID := util.GenerateLeaderElectionID(types.KubeVelaName+"-"+leaderElectionName, controllerArgs.IgnoreAppWithoutControllerRequirement)
```

Comment:
I wonder if it will cause incompatible?

Reply:
I think it's a leader-election configmap name and be compatible.

Reply:
if leaderElectionName is empty, the "-" should be omitted